src/utils.js

```diff
@@ -8,3 +8,19 @@ exports.findHighest = (...numbers) => {
     return highest;    
 };
 
+
+exports.findLowest = (...numbers) => {
```

Comment:
Lets ensure the following for this function

- at least one number must be provided. if no numbers are provided (e.g `findLowest()`, an error  should be thrown)
- if any of the provided arguments is not a number, you should throw an error

Once these are in place, it should be guaranteed that the lowest can not be null or infinity or NaN

Reply:
ok